gadget/install/install.go

```diff
@@ -239,9 +273,23 @@ func Run(model gadget.Model, gadgetRoot, kernelRoot, bootDevice string, options
 		created, err = CreateMissingPartitions(diskLayout, laidOutBootVol, opts)
 	})
 	if err != nil {
-		return nil, fmt.Errorf("cannot create the partitions: %v", err)
+		return nil, nil, 0, "", fmt.Errorf("cannot create the partitions: %v", err)
 	}
 
+	return allLaidOutVols, created, diskLayout.SectorSize, laidOutBootVol.Name, nil
+}
+
+// Run creates partitions, encrypts them when expected, creates
+// filesystems, and finally writes content on them.
+func Run(model gadget.Model, gadgetRoot, kernelRoot, bootDevice string, options Options, observer gadget.ContentObserver, perfTimings timings.Measurer) (*InstalledSystemSideData, error) {
+	// Step 1: create partitions
+	allLaidOutVols, created, sectorSize, bootVolGadgetName, err :=
```

Comment:
(nitpick^3) I prefer the longer lines here over spliting into two :)

Reply:
see comment above about how to change the return values of this

Reply:
Changed now. Re: lines, for me it makes it more readable, but of course it is a matter of opinion,